### ISSUE TITLE
Would the IAO be the appropriate place to insert a "datum status" concept

### ISSUE THREAD

**Issue by Public-Health-Bioinformatics** [information-artifact-ontology/IAO#192]
This is from the European Nucleotide Archive website:
"The International Nucleotide Database Collaboration (INSDC) have developed a standardised missing/null value reporting language to be used where a value of an expected format for sample metadata reporting can not be provided." - [](http://www.ebi.ac.uk/ena/about/missing-values-reporting)

<img width="300" alt="datum_status" src="https://cloud.githubusercontent.com/assets/10779910/22054926/3d47e054-dd0d-11e6-83ac-f61f12bd7c5b.png">

Would IAO be a good home for these guys?  I would actually propose a few changes, e.g. add one more category 'in process' if there was some commitment to deliver the information; as well a "recorded" status would in theory complete these rough metadata states.  Right now the GenEpiO ontology has these terms but they are best positioned in a more general ontology.

**Comment by alanruttenberg** [comment]
Yes. See the term status instances already in IAO (ontology metadata). We might want to talk about the label for the class - datum status is rather broad. From an IAO point of view, status labels are information about what they are the status of.

**Comment by Public-Health-Bioinformatics** [comment]
Perhaps "datum retrieval state" then? I realize now that reporting that a datum instance is in some state like this would depend on a storage or intermediary system attempting to fetch a datum instance and finding the instance is in one of these metadata states instead.

**Comment by Public-Health-Bioinformatics** [comment]
I've put up a pull request for your consideratio

**Comment by Public-Health-Bioinformatics** [comment]
About term status as instances ... I get how one connects a class via "has curation status" to an instance like "requires discussion".  But I like the idea of keeping data request state a class since it is likely to be further subclassed in the future, e.g. "data obfuscated" could have subclasses "data obfuscated by randomization" and "data obfuscated by scaling" etc. But we could punn these to have owl:NamedInstance ids for those who want.  

Either way, I think we can use "**has quality**" relation between "data item" instances and a "data request state"?  Or must we introduce a "has data request state" (domain data item, range data request state)?

**Comment by alanruttenberg** [comment]
can't use 'has quality' at the moment, since only independent continuants can have qualities, so a different property. There is an ongoing discussion advocating that generically dependent continuants can have dependents themselves, but consensus has not been reached.

I see the desire to be able to subdivide later. I'm not sure about whether I think its a good idea, since it is an easy path to moving things that might be more properly represented elsewhere. 
For example, a more transparent way to encode obfuscation would be to have a data transformation type and represent that the data is the output of such a process.
